build-packages.cmd

```diff
@@ -60,6 +60,15 @@ if NOT [!ERRORLEVEL!]==[0] (
   exit /b 1
 )
 
+rem Build the TestHost package
+set __msbuildArgs="%__ProjectDir%\src\.nuget\Microsoft.NETCore.TestHost\Microsoft.NETCore.TestHost.builds" !allargs!
```

Comment:
I don't see any reason not to switch to calling build on packages.builds. @maririos and @Priya91 are making changes to these scripts right now, what do you guys think (build-packages is going to look very different as soon as Priya's change goes in anyways, so waiting might be the right move here)?